plugins/bufbuild/connect-query/v0.1.0/buf.plugin.yaml

```diff
@@ -0,0 +1,21 @@
+version: v1
+name: buf.build/bufbuild/connect-query
+plugin_version: v0.1.0
+source_url: https://github.com/bufbuild/connect-query
+description: Generates client stubs for calling Connect services with TanStack Query.
+deps:
+  - plugin: buf.build/bufbuild/connect-web:v0.6.0
```

Comment:
Looking at the package.json for @bufbuild/connect-query, I see it depends on connect-web. This will also pull in protobuf-es (connect-web depends on it).